src/dynimage.rs

```diff
@@ -878,30 +878,76 @@ impl DynamicImage {
         dynamic_map!(*self, ref p => imageops::flip_vertical(p))
     }
 
+    /// Flip this image vertically in place
+    pub fn flipv_in_place(&mut self) {
+        dynamic_map!(*self, ref mut p, imageops::flip_vertical_in_place(p))
+    }
+
     /// Flip this image horizontally
```

Comment:
Might be worth suggesting `apply_rotation(Orientation::FlipH)` as an in-place alternative of the same operation